### PR TITLE
Migrate to Swift3

### LINKED ISSUE #41: Migrate to Swift3

### PR DESCRIPTION
**Pull request by stel**
fixes #41

### CODE REVIEW

File: RealmConverter/Importer/DataImporter.swift [hunk 1]
```diff
@@ -75,59 +75,61 @@ public class DataImporter: NSObject {
...
-                let ivar = objc_property_attribute_t(name: "V".cStringUsingEncoding(NSUTF8StringEncoding), value: property.name.cStringUsingEncoding(NSUTF8StringEncoding)!)
+                let ivar = objc_property_attribute_t(name: "V".cString(using: .utf8), value: property.name.cString(using: .utf8)!)
                 let attrs = [type, ivar]
                 class_addProperty(cls, property.originalName, attrs, 2)
                 
                 let imp = imp_implementationWithBlock(unsafeBitCast({ () -> Bool in
                     return true
-                    } as @convention(block) () -> (Bool), AnyObject.self))
-                class_addMethod(cls, #selector(NSObjectProtocol.respondsToSelector(_:)), imp, "b@::")
+                    } as @convention(block) () -> (Bool), to: AnyObject.self))
```
Comment (jpsim) [inline RealmConverter/Importer/DataImporter.swift]: indentation is off here

File: RealmConverter/Importer/DataImporter.swift [hunk 2]
```diff
@@ -75,59 +75,61 @@ public class DataImporter: NSObject {
...
+                class_addMethod(cls, #selector(NSObjectProtocol.responds(to:)), imp, "b@::")
             }
             
             objc_registerClassPair(cls);
             
             let imp = imp_implementationWithBlock(unsafeBitCast({ () -> NSArray in
-                return schema.properties.filter { !$0.optional }.map { $0.originalName }
-                } as @convention(block) () -> (NSArray), AnyObject.self))
+                return schema.properties.filter { !$0.optional }.map { $0.originalName } as NSArray
+                } as @convention(block) () -> (NSArray), to: AnyObject.self))
```
Comment (jpsim) [inline RealmConverter/Importer/DataImporter.swift]: indentation is off here

**Comment by stel** [comment]
All tests passed but unfortunately it seems like there is no way to release it on Cocoapods because of it depends on CSwiftV 0.0.7 which is not available from pods. I've created https://github.com/Daniel1of1/CSwiftV/issues/34.

For now RealmConverter with Swift3 support can be used via CocoaPods like this:

```
...
pod 'CSwiftV', :git => 'https://github.com/Daniel1of1/CSwiftV.git', :tag => '0.0.7'
pod 'RealmConverter', :git => 'https://github.com/realm/realm-cocoa-converter.git', :branch => 'do-swift3'
...
```

**Comment by stel** [comment]
Linting failed because of the warnings, everything else seems fine so merging and releasing 0.4.0.

**Pull request merged**